cni-plugin/pkg/install/install_test.go

```diff
@@ -158,6 +216,32 @@ PuB/TL+u2y+iQUyXxLy3
 			Fail(fmt.Sprintf("Failed to write k8s CA file for test: %v", err))
 		}
 
+		// Create namespace file for token refresh
+		k8sNamespace := []byte("kube-system")
+		var namespaceFile = fmt.Sprintf("%s/serviceaccount/namespace", tempDir)
+		err = ioutil.WriteFile(namespaceFile, k8sNamespace, 0755)
+		if err != nil {
+			Fail(fmt.Sprintf("Failed to write k8s namespace file: %v", err))
+		}
+
+		// Create calico-node service account
+		serviceAccount := &v1.ServiceAccount{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: "calico-node",
+			},
+		}
+		_, err = createKubernetesClient().CoreV1().ServiceAccounts("kube-system").Create(context.Background(), serviceAccount, metav1.CreateOptions{})
+		if err != nil {
```

Comment:
You can use `Expect(err).NotTo(HaveOccurred())` for all of these error checks in test code

Reply:
Adapted accordingly.